device/mellanox/x86_64-mlnx_msn4700-r0/Mellanox-SN4700-A96C8V8/hwsku.json

```diff
@@ -0,0 +1,340 @@
+        },
+        "Ethernet129": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet130": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet131": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet136": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet137": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet138": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet139": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet144": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet145": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet146": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet147": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet152": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet153": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet154": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet155": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet160": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet161": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet162": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet163": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet168": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet169": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet170": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet171": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet176": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet177": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet178": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet179": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet184": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet185": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet186": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet187": {
+            "default_brkout_mode": "4x25G(4)[10G,1G]"
+        },
+        "Ethernet192": {
+            "default_brkout_mode": "2x200G[100G,50G,40G,25G,10G,1G]"
```

Comment:
Is there a way to initialize this as 2x100G instead of 2x200 and later setting the speed?

Reply:
Please use this as a referene
```
https://github.com/sonic-net/sonic-buildimage/pull/9278
```

Reply:
Updated